src/components/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import "../../styling/Header.css"
+
+function Header() {
+    return <header className="header">
+        <div className="header-titles">
```

Comment:
Consider using more semantic elements in your code to improve readability and accessibility